library/src/main/java/com/pixplicity/sharp/Sharp.java

```diff
@@ -1724,7 +1724,7 @@ private void finishGradients() {
                     positions[i] = gradient.mPositions.get(i);
                 }
                 if (colors.length == 0) {
-                    Log.w(TAG, "bad gradient, id=" + gradient.mId);
+                    Log.d(TAG, "bad gradient, id=" + gradient.mId);
```

Comment:
I disagree that this should be a debug message. The gradient is invalid, and a warning should be logged.